Update README.md (#48)
pureblazor/components: README.md

> This component library is pre-release software.

These components are native Blazor UI components that are free to use for any Blazor project. The components also work seamlessly with [PureBlazor CMS](https://pureblazor.com).

# Features

- **Native Blazor** - The components are built for Blazor. They are not a wrapper around a JavaScript library.
- **Blazing Fast** - The components are fast on every platform.
- **Headless Mode** - The components have a fully customizable UI. You may disable all styles with a single property.

> - **Tailwind Compatible** - The components have  built-in styles that you may automatically merge your Tailwind classes.

Reviewer: I think there's a missing word here FYI

> - **Tailwind Compatible** - The components have  built-in styles that you may automatically merge your Tailwind classes.

Reviewer: ```suggestion
- **Tailwind Compatible** - The components have  built-in styles that automatically merge with your Tailwind classes.
```
